app/views/alchemy/pages/show.rb

```diff
@@ -75,7 +75,11 @@ def title
       end
 
       def page_title
-        @page.title
+        [
+          foyer? ? nil : @page.title,
+          t('global.navigation.exhibitions', default: 'Exhibitions'),
```

Comment:
Let's remove the defaults to `I18n.t` in the changes on this branch so that tests fail if the key does not exist in the localisations, and the localised English is used as the fallback for other languages.